internal/hooks/v0hooks/manager.go

```diff
@@ -215,35 +132,36 @@ func (o *Manager) invokeHook(
 				HTTPStatus: httpCode,
 				Message:    err.Error(),
 			}
-
 			return httpError
 		}
 		return nil
 	}
-	return nil
 }
 
 func (o *Manager) runHook(
 	r *http.Request,
 	conn *storage.Connection,
 	hookConfig conf.ExtensibilityPointConfiguration,
 	input, output any,
-) ([]byte, error) {
+) error {
 	ctx := r.Context()
 
 	logEntry := observability.GetLogEntry(r)
 	hookStart := time.Now()
 
-	var response []byte
 	var err error
-
 	switch {
-	case strings.HasPrefix(hookConfig.URI, "http:") || strings.HasPrefix(hookConfig.URI, "https:"):
-		response, err = o.runHTTPHook(r, hookConfig, input)
+	case strings.HasPrefix(hookConfig.URI, "http:") ||
+		strings.HasPrefix(hookConfig.URI, "https:"):
+		err = o.v0http.Dispatch(ctx, hookConfig, input, output)
+
 	case strings.HasPrefix(hookConfig.URI, "pg-functions:"):
-		response, err = o.runPostgresHook(ctx, conn, hookConfig, input, output)
+		err = o.v0pgfunc.Dispatch(ctx, hookConfig, conn, input, output)
+
 	default:
-		return nil, fmt.Errorf("unsupported protocol: %q only postgres hooks and HTTPS functions are supported at the moment", hookConfig.URI)
+		return fmt.Errorf(
+			"unsupported protocol: %q only postgres hooks and HTTPS functions"+
```

Comment:
is the "+" necessary here or just for readability sake?

Reply:
Not necessary but I prefer it for these really long lines.